includes/abandoned-signups.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+ * Set up a user taxonomy to track users who were created during the PMPro
+ * checkout process but did not complete the checkout.
+ *
+ * @since TBD
+ */
+function pmpro_abandoned_signups_taxonomy() {
```

Comment:
I'm thinking it's good to get picky about the naming here. Our first priority is to flag users that never complete checkout so we can delete them after a few days, but in reality this taxonomy can be used later for abandoned cart functionality, which would be really cool.

Reply:
So I'm thinking something more like "pmpro_checkout_user_status" and having terms like "pending", "abandoned", and "completed".

Pending = user created at checkout
Abandoned = maybe updated via cron X days later
Completed = updated when checkout is completed

Reply:
Let's merge pending and abandoned into an "incomplete" status. This avoids the need for a cron, but doesn't have the loaded "abandoned" language that might be problematic if it "gets out" in future updates.

Reply:
Let's merge pending and abandoned into an "incomplete" status. This avoids the need for a cron, but doesn't have the loaded "abandoned" language that might be problematic if it "gets out" in future updates.